src/bot/TelegramConnector.js

```diff
@@ -44,10 +44,27 @@ export default class TelegramConnector
   }
 
   getUniqueSessionKey(body: TelegramRequestBody): string {
-    if (body.message !== undefined) {
-      return `${body.message.from.id}`;
-    } else if (body.callback_query !== undefined) {
+    if (body.message) {
+      return `${body.message.chat.id}`;
+    } else if (body.edited_message) {
```

Comment:
does the sequence of these if conditions matter?

Reply:
it doesn't matter